test/rest.test.js

```diff
@@ -1359,6 +1359,112 @@ describe('strong-remoting-rest', function() {
           });
       });
 
+      it('should allow customized xml root element', function(done) {
+        var method = givenSharedStaticMethod(
+          function bar(cb) {
+            cb(null, {a: 1, b: 2});
+          },
+          {
+            returns: { arg: 'data', type: 'object', root: true,
+            xml: { wrapperElement: 'foo'} },
+            http: { path: '/' }
+          }
+        );
+        request(app).get(method.classUrl)
+          .set('Accept', 'application/xml')
+          .set('Content-Type', 'application/json')
+          .send()
+          .expect('Content-Type', /xml/)
+          .expect(200, function(err, res) {
+            expect(res.text).to.equal(
+              '<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n' +
+              '<foo>\n  ' +
+                '<a>1</a>\n  ' +
+                '<b>2</b>\n' +
+              '</foo>');
+            done(err, res);
+          });
+      });
+
+      it('should allow xml decorations to be disabled', function(done) {
+        var method = givenSharedStaticMethod(
+          function bar(cb) {
+            cb(null, {a: 1, b: 2});
+          },
+          {
+            returns: { arg: 'data', type: 'object', root: true,
+            xml: { declaration : false } },
+            http: { path: '/' }
+          }
+        );
+        request(app).get(method.classUrl)
+          .set('Accept', 'application/xml')
+          .set('Content-Type', 'application/json')
+          .send()
+          .expect('Content-Type', /xml/)
+          .expect(200, function(err, res) {
+            expect(res.text).to.equal(
+              '<response>\n  ' +
+                '<a>1</a>\n  ' +
+                '<b>2</b>\n' +
+              '</response>');
+            done(err, res);
+          });
+      });
+
+      it('should allow string results to output as xml', function(done) {
+        var method = givenSharedStaticMethod(
+          function bar(cb) {
+            var stringResult = 'a quick brown fox jumps over the lazy dog';
+            cb(null, stringResult);
+          },
+          {
+            returns: {root: true},
+            http: { path: '/' }
+          }
+        );
+        request(app).get(method.classUrl)
+          .set('Accept', 'application/xml')
+          .set('Content-Type', 'application/json')
+          .send()
+          .expect('Content-Type', /xml/)
+          .expect(200, function(err, res) {
+            expect(res.text).to.equal(
+              '<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n' +
+              '<response>a quick brown fox jumps over the lazy dog' +
```

Comment:
Please modify the test to use a custom root element, e.g. `<text>` instead of `<response>`.